src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    private final int index = new Random().nextInt(Color.values().length);
```

Comment:
index must be generated in method
```suggestion
    private Random random = new Random();
```